File: java/sqlbuilder/DataaccessException.java
package com.clickability.cms.dataaccess.sqlbuilder;

/**
 * Same as SQLException except that it is Runtime so that
 * it can be used in lambda expressions.
 * 
 * @author roystyan
 *
 */
public class DataaccessException extends RuntimeException {
	private static final long serialVersionUID = 1L;
	
	public DataaccessException(Exception cause) {
		super(cause);
	}
	
	public DataaccessException(String message) {
		super(message);
	}
}

File: src/main/kotlin/queryBuilder/com/inunison/lib/queryBuilder/DataaccessException.kt
package com.clickability.cms.dataaccess.sqlbuilder

/**
 * Same as SQLException except that it is Runtime so that
 * it can be used in lambda expressions.
 *
 * @author roystyan
 */
class DataaccessException : RuntimeException {

    constructor(cause: Exception) : super(cause) {}

    constructor(message: String) : super(message) {}

    companion object {
        private val serialVersionUID = 1L
    }
}
